cypress/integration/login.spec.js

```diff
@@ -1,13 +1,31 @@
-import { ACCESSIBILITY_ADMIN_DEV } from '../../src/constants/jobCodes'
+import { ACCESSIBILITY_ADMIN_DEV } from '../../src/constants/jobCodes';
 
+const maxAttempts = 3;
 describe('Logging in', () => {
-  it('logs in with okta', () => {
-    cy.login();
-    cy.location('pathname', { timeout: 20000 }).should('equal', '/');
-  });
+  it(
+    'logs in with okta',
+    {
+      retries: {
+        runMode: maxAttempts - 1, // 2 retries when running from `cypress run` (3 total attempts)
+        openMode: 0 // 0 retries when running from `cypress open` (1 total attempt)
+      }
+    },
+    () => {
+      // Get the current number of retries and sleep 60s before running the test to make sure the One-Time-Password is new
```

Comment:
Comment doesn't match the actual sleep time after the change to 30s. It might be easier to make the sleep time into a constant and reference that in the comment (or just remove the specific time duration in the comment).

Reply:
Oops! Thought I caught this one too when I switched to 30. I like the second approach

Reply:
1cf2a25fd